modules/aws_autoscaling/main.tf

```diff
@@ -6,7 +6,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~> 3.0"
+      version = "~> 5.0"
```

Comment:
Another major change. Just be sure no breaking changes.

Reply:
I think we've got a good handle on the breaking changes for the AWS provider, I've reviewed the breaking changes listed in each one of the 4.x and 5.x `aws` provider releases as well as both the 4.x and 5.x upgrade guides. Both of those agreed on breaking changes with only a few items found in one but not the other. There were many 4.x releases but in many cases the breaking changes affect resources or data sources that do not intersect with features that we are using.